serve-seed.ejs

```diff
@@ -5,6 +5,7 @@
     <!-- All these libraries will be bundled in the proxy when we're ready -->
     <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
     <script src="https://unpkg.com/jschannel"></script>
+    <script src="https://unpkg.com/urijs"></script>
     <script src="<%= seedUrl %>/apiary-proxy.js"></script>
     <script>
       Apiary.buildChannel('<%= seedOrigin %>', '<%= seedScope %>');
```

Comment:
Could you elaborate more what you mean by `keep`?

If you mean expose it to the public, I'm doing that just for testing purposes.